### DISCUSSION TITLE
Which version is your recommendation?

### DISCUSSION THREAD

**Discussion by physicshinzui** [HITS-MCM/gromacs-ramd#26] [Q&A] [answered]
Let me just confirm. 

I've fetched RAMD codes via `git clone https://github.com/HITS-MCM/gromacs-ramd.git`
And I've successfully compiled it. 
Is the version given by `git clone https://github.com/HITS-MCM/gromacs-ramd.git` a recommended version to use? 

Thank you in advance.

**Comment by BerndDoser** [comment, marked as answer]
I clearly recommend the latest [release](https://github.com/HITS-MCM/gromacs-ramd/releases). The version via `git clone` contains the current development state. You can also get the releases using `git checkout <version-tag>`.

**Reply by gsaha322** [reply]
Can you please provide details how the RAMD codes can be patched with GROMACS as I am getting this error as shown in the screenshot.
![Screenshot 2023-01-08 222407](https://user-images.githubusercontent.com/122028806/211208989-bd02d8b5-25a0-40f3-8091-10a293e32848.jpg)

**Reply by BerndDoser** [reply]
RAMD is not available in the official GROMACS, please see #32.
